src/BinaryChannel.hpp

```diff
@@ -57,6 +61,10 @@ class BinaryWriter : public Writer
     virtual void writeUInt24(std::string name, uint32_t value);
     virtual void writeUInt32(std::string name, uint32_t value);
 
+    void writeHex8(std::string name, uint8_t value);
+    void writeHex16(std::string name, uint16_t value);
+    void writeHex32(std::string name, uint32_t value);
```

Comment:
```suggestion
    void writeHex8(std::string name, uint8_t value) override;
    void writeHex16(std::string name, uint16_t value) override;
    void writeHex32(std::string name, uint32_t value) override;
```

Since you started with override above it should be in all overridden function.

Reply:
I forgot to do that before committing